### PR TITLE
Add the Memcached plugin

### PR DESCRIPTION
**Pull request by jvanbaarsen**
Fixes: #88

### CODE REVIEW

File: lib/tasks/seed.rake [hunk 1]
```diff
@@ -7,6 +7,11 @@ namespace :intercity do
                     create: "dokku redis:create %app_name%_redis",
                     link: "dokku redis:link %app_name%_redis %app_name%",
                     backup: "dokku redis:export %app_name%_redis" }},
+      { name: "Memcached", active: true,
+        commands: { install: "sudo dokku plugin:install https://github.com/dokku/dokku-memcached.git memcached",
+                    create: "dokku memcached:create %app_name%_memcached",
+                    link: "dokku redis:link %app_name%_memcached %app_name%",
```
Comment (michiels) [inline lib/tasks/seed.rake]: redis:link?

Comment (jvanbaarsen) [inline lib/tasks/seed.rake]: Ok, I'm a retard.. I did the same with `create` :/

Comment (michiels) [inline lib/tasks/seed.rake]: The create is correct, right? :)

Comment (jvanbaarsen) [inline lib/tasks/seed.rake]: Yeah after I changed it haha

Comment (michiels) [inline lib/tasks/seed.rake]: :+1:

**Pull request merged**